doc/api/index.rst

```diff
@@ -238,6 +238,7 @@ Use :func:`pygmt.datasets.load_sample_data` instead.
     datasets.load_fractures_compilation
     datasets.load_hotspots
     datasets.load_japan_quakes
+    datasets.load_map_tiles
```

Comment:
I'm unsure about the name. Should it be called `load_map_tiles` or `load_tile_maps`?

Reply:
I was thinking about the 2nd part of #2115, and had an idea about making a `fig.tilemap` method instead of complicating `fig.basemap` further. In that case, `load_tile_maps` might actually work better?

Reply:
Are both "map tiles" and "tile maps" grammatically correct?

Reply:
I think both should be ok. At least I know there's [Web Map Tile Service (WMTS)](https://en.wikipedia.org/wiki/Web_Map_Tile_Service) and [Tile Map Service (TMS)](https://en.wikipedia.org/wiki/Tile_Map_Service)

Reply:
I'm starting to lean towards `load_tile_maps` or `load_tile_map`. Should there be an `s` or no `s`?

Reply:
I prefer `load_tile_map` without `s`.

Reply:
Ok, renamed to `load_tile_map` in 3dc8846585df4b4e74ae2dd7887b979d1bac1bd2